ngraph/frontend/onnx_import/src/op/constant.cpp

```diff
@@ -33,8 +34,21 @@ namespace ngraph
                     inline std::shared_ptr<default_opset::Constant>
                         __make_ng_constant(const element::Type& type, const Tensor& tensor)
                     {
-                        return std::make_shared<default_opset::Constant>(
-                            type, tensor.get_shape(), tensor.get_data<T>());
+                        std::shared_ptr<default_opset::Constant> constant{nullptr};
+                        try
+                        {
+                            constant = std::make_shared<default_opset::Constant>(
+                                type, tensor.get_shape(), tensor.get_data<T>());
+                        }
+                        catch (const ngraph::ngraph_error& exc)
+                        {
+                            NGRAPH_WARN << "Could not create an nGraph Constant for an ONNX "
+                                           "Constant node. Detailed error:\n"
```

Comment:
OK, there was an error, now what?
If there is a warning it should tell what happened(Constant with scalar 0 has been created) otherwise it can be confusing.

Reply:
It's just to have the same behavior as we have for empty initializers (graph.cpp:93)